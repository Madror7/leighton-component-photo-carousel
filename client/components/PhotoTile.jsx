import React from 'react';

const PhotoTile = props => (
  <div className="photo-tile" onClick={(() => passId(props)).bind(this)}>
    <img src={props.link}></img>
  </div>
);

const passId = (props) => {
  props.openModal(props.id);
}

export default PhotoTile;
