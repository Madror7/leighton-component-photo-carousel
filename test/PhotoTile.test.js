/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import PhotoTile from '../client/components/PhotoTile';

describe('PhotoTile', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<PhotoTile debug />);

    expect(component).toMatchSnapshot();
  });
});
