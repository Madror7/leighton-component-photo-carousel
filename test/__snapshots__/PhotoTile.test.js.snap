// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoTile should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
